pychunkedgraph/app/segmentation/common.py

```diff
@@ -686,11 +686,12 @@ def handle_leaves(table_id, root_id):
         ).T
     else:
         bounding_box = None
-
+    stop_layer = int(request.args.get('stop_layer', 1))
     # Call ChunkedGraph
     cg = app_utils.get_cg(table_id)
     atomic_ids = cg.get_subgraph_nodes(
```

Comment:
This will return a dictionary 
stop_layer -> node ids
https://github.com/seung-lab/PyChunkedGraph/blob/master/pychunkedgraph/backend/chunkedgraph.py#L3313

`stop_layer=2` this can be keyed with `stop_layer` but above that it is not that easy because of skip connections.